routemaster.rb

```diff
@@ -2,4 +2,30 @@ module Routemaster
   def self.now
     (Time.now.utc.to_f * 1e3).to_i
   end
+
+  DEFAULTS = {
+    redis_pool_size: 1
+  }.freeze
```

Comment:
doesn't the fact it's a constant already issue at least a warning on modification?